### DISCUSSION TITLE
SSD365VSAddins in Power Platform admin enabled DEV VM?

### DISCUSSION THREAD

**Discussion by LakshmiVijay79** [shashisadasivan/SSD365VSAddIn#64] [Q&A]
Hi Shashi,
SSD365VS addins - while installing we need to locate the Dynamics config file but not sure where it is located in Power platform admin enabled D365 F&O Dev VM

**Comment by shashisadasivan** [comment]
> Hi Shashi, SSD365VS addins - while installing we need to locate the Dynamics config file but not sure where it is located in Power platform admin enabled D365 F&O Dev VM

Hi @LakshmiVijay79 , i havent tested this with UDE, so it will be VS2022 as well i assume?
However, the config file is usually saved under C:\Users\<username>\Documents\Visual Studio Dynamics 365\DynamicsDevConfig.xml

If you can see that in your machine, then the best way would be to install it via the install script (instead of manually doing it)
use Option1: [How to install the extension](https://github.com/shashisadasivan/SSD365VSAddIn?tab=readme-ov-file#how-to-install-the-extension)

If this doesnt work, then let me know if you are happy to jump into a call to sort this out.

**Comment by LakshmiVijay79** [comment]
thanks Shashi for the quick response. I tried to find out the same
folder(Dyanmics 365) but in that no config file found.
Earlier, i did the same for LCS VMs and installed manually.
Also note that, when I try to run the Power shell script to install,
multiple errors occurred.



Thanks,
Vijay DP

**Reply by shashisadasivan** [reply]
Hi Vijay, sorry for the extremely late reply.
I haven't had a chance to play with this in VS2022 yet. 
happy to connect and look through it with you if you wish

**Reply by shashisadasivan** [reply]
@LakshmiVijay79 I have started upgrading the code for VS2022. Please follow issue #66 
Let me know if any functionality fails in the process

**Comment by LakshmiVijay79** [comment]
thank you. Also noticed that while generating the labels, we are getting
best practice warning saying that the label id is not aligned.
EX: LabelId is not same case as the label. Kindly check it.


Thanks,
Vijay DP

**Reply by shashisadasivan** [reply]
Thanks, Could you put an example of the Generated Id and the label please
The one that gets written to the file. Thanks
Maybe open a different issue for this ! :)

**Comment by shashisadasivan** [comment]
Upgraded to VS2022 as per #66

**Comment by LakshmiVijay79** [comment]
Hi Shashi,
As per my analysis , this mod wont work as we dont have dynamics config
file inside UDE VMs. Please check


Thanks,
Vijay DP

**Reply by shashisadasivan** [reply]
Thanks @LakshmiVijay79 
Once i start on a UDE environment i will update this. Until then i will create this into an issue.

**Comment by LakshmiVijay79** [comment]
thank you. Any other way, I can import as an XPP to generate labels?


Thanks,
Vijay DP

**Reply by shashisadasivan** [reply]
Not at this stage. there might be some other standalone tools created for this
